src/react-components/remix-scene-dialog.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { FormattedMessage } from "react-intl";
+
+import styles from "../assets/stylesheets/remix-scene-dialog.scss";
+import DialogContainer from "./dialog-container";
+import { getPlatform, fetchRelease, releasesLink } from "../utils/spoke-download-link";
+
+const Steps = {
+  DownloadSpoke: "download-spoke",
+  DownloadScene: "download-scene",
+  OpenScene: "open-scene"
+};
+
+export default class RemixSceneDialog extends Component {
+  static propTypes = {
+    sceneUrl: PropTypes.string,
```

Comment:
i believe you are using `spokeUrl` elsewhere, i'd probably call this `sceneProjectUrl` everywhere as we are now using that term in the API